main.py

```diff
@@ -19,22 +18,153 @@ def get_horsepower(self):
     def get_classification(self):
         '''Function to determine classification'''
         if self._horsepower <= 130:
-            return 'Compact'
+            return 'Compact Vehicle'
         if self._horsepower <= 170:
-            return 'Midsize'
-        return 'Fullsize'
+            return 'Midsize Vehicle'
+        return 'Fullsize Vehicle'
+
+    def __repr__(self):
+        '''Function to get printable representation'''
+        return 'Vehicle with' + str(self._horsepower) + 'horsepower'
```

Comment:
repr should ideally be more specific in showing how to reinitialize the class or current state